ochopod/frameworks/marathon.py

```diff
@@ -55,6 +55,8 @@ class Marathon(Binding):
         - *ochopod_debug*: turns debug logging on if set to "true".
         - *ochopod_namespace*: namespace as dot separated tokens (e.g "my-app.staging"), defaulted to "marathon".
         - *ochopod_port*: pod control port on which we listen for HTTP requests, defaulted to 8080.
+        - *ochopod_zk*: location of ZK ensemble. The string must in the format of zk://<ip>:<port>,
```

Comment:
just one tiny thing: please state in the comment this is optional and will be used at a last-resort mostly for debugging